tests/baselines/reference/tsbuild/javascriptProjectEmit/initial-build/loads-js-based-projects-with-non-moved-json-files-and-emits-them-correctly.js

```diff
@@ -285,7 +285,7 @@ module.exports = x;
       },
       "./obj.json": {
         "version": "2151907832-{\n    \"val\": 42\n}",
-        "signature": "-6323167306-export declare const val: number;\r\n",
+        "signature": "-5546159834-export const val: number;\r\n",
```

Comment:
So! Fun fact: We parse json documents essentially as `export = { val: 42 }`. Previous to this change, we'd serialize it to
```ts
declare namespace export_ {
    export declare const val: number;
}
export = _export;
```
then inline it to
```ts
export declare const val: number;
```
Now, we first make
```ts
declare namespace export_ {
    const val: number;
}
export = _export;
```
since we can omit the export and declare modifiers, and inline to
```ts
const val: number;
```
which we then patch an `export` modifier back onto (since we're inlining a namespace where everything was implicitly exported):
```ts
export const val: number;
```
so our frugality with the `export` modifier has, in this case, also caused us to be more frugal with the `declare` modifier, too. This should be fine, since everything in a `.d.ts` file is already implicitly `declare`'d anyway.